packages/theming/atlas/tests/e2e/allScreens.spec.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require("fs");
+const { join, resolve } = require("path");
+const cwd = process.cwd();
+const xmlParser = require("fast-xml-parser");
+
+// For every module mx exports a folder, and in that folder we have bunch of xml's
+// which might contains URL field
+const individualPageFolders = join(cwd, "tests/testProject/deployment/web/pages/en_US");
+
+// Make an async function that gets executed immediately
+function* getFiles(dir) {
+    const dirents = fs.readdirSync(dir, { withFileTypes: true });
```

Comment:
Nit: what is a `dirents`? Based on the code below, it seems to be either a directory or a file.

Reply:
Directory entries

Reply:
I think it should be something anybody could understand :)

Reply:
Fixed